metadata_backend/helpers/schemas/bp_bpdataset.json

```diff
@@ -1,8 +1,8 @@
 {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
this line is not required

Reply:
It's not required, but for future reference, it's good to know which schema version we have. It was not so easy (without prior JSON schema knowledge) to know that most of the schemas were Draft7. If they release new versions and the parsers update their default version, our tests will keep passing, because the parsers should detect the schema version from that `$schema` field.

Reply:
ok, then we can leave it.